lib/helpers.rb

```diff
@@ -92,8 +93,13 @@ def write_file(directory,filename,output)
       ###
       ### Helper to upload to kenna api
       ###
+<<<<<<< Updated upstream
       def upload_file_to_kenna_connector(connector_id, api_host, api_token, filename)
         # optionally upload the file if a connector ID has been specified 
+=======
```

Comment:
Are these changes expected?

Reply:
Good point... missed those when reading through this.  Thoughts @tmgerhart?